VirtoCommerce.Storefront.Model/Catalog/CategorySearchCriteria.cs

```diff
@@ -42,6 +42,7 @@ public CategorySearchCriteria(Language language, NameValueCollection queryString
 
         private void Parse(NameValueCollection queryString)
         {
+            IsFuzzySearch = queryString.Get("fuzzy").EqualsInvariant(bool.FalseString);
```

Comment:
Why comparing with False? Thus we set IsFuzzySearch=true only when passing fuzzy=false.

Reply:
Ooops, sorry, late night PR :)